gatk-launch

```diff
@@ -86,6 +86,7 @@ def main(args):
             print("     LOCAL:   run using the in-memory spark runner")
             print("     SPARK:   run using spark-submit on an existing cluster ")
             print("              --sparkMaster must be specified")
+            print("              --sparkSubmit may be specified to control the Spark submit command")
```

Comment:
I think this needs clearer wording, it sounds like it's giving options to the spark-submit program, not specifying the path to it.  Maybe "--sparkSubmitPath may be used to specify the path to the spark-submit program"?